### PR TITLE
fix(footer): align left footer buttons with right theme switcher

### PR DESCRIPTION
**Pull request by sverps**
This adds a background to the fixed footer elements, but only on mobile. #123

### CODE REVIEW

**Comment by carletex** [comment]
This is related to https://github.com/scaffold-eth/se-2/issues/123

I'd say: 

- Let's remove the background: Not a fan of the magic div with a magic number for the height.
- Use `md` instead of `lg`
- `mb-10` should be enough (instead of `mb-16`)

A couple of more fixes (existing errors):
- Could you vertically center the button with the theme switcher?
image

- Make the "Fork me...." to wrap.
image

Thanks @sverps !!

**Comment by sverps** [comment]
Thanks for the remarks :+1: 
I'll clean it up a bit when I get some more time.

**Comment by sverps** [comment]
Removed the background:
image

As for the wrapping of the "Fork me", even on devices as small as 360px, it still renders decent, so not sure it really needs wrapping (and I couldn't get flex-wrap to work for some reason... :disappointed: )

File: packages/nextjs/components/Footer.tsx [hunk 1]
```diff
@@ -15,9 +15,9 @@ export default function Footer() {
   const configuredNetwork = getTargetNetwork();
 
   return (
-    <div className="min-h-0 p-5 flex justify-between items-center flex-col sm:flex-row gap-4">
+    <div className="min-h-0 p-5 flex justify-between items-center flex-col sm:flex-row gap-4 mb-10 lg:mb-0">
       <div>
-        <div className="flex space-x-2 fixed z-10 m-4 bottom-0 left-0">
+        <div className="flex items-center space-x-2 fixed z-20 h-16 bottom-0 left-4">
```
Comment (carletex) [inline packages/nextjs/components/Footer.tsx:20]: We shouldn't use magic numbers (like h-16, left-4)

Height should be a result of the element size + paddings. And we shouldn't move elements with "left / right" props (only `0`)

**Review by carletex — CHANGES_REQUESTED** [review]
left a code review comment. Thanks

**Comment by sverps** [comment]
Had to put all fixed elements inside the same fixed parent (rather than positioning them separately) in order to be able to align them.

The pointer-events need to be passed through the parent container to access the footer links that are rendered at a lower zIndex.

Additionally, `mb-11` looked like a slightly better choice than `mb-10`:
image

**Comment by carletex** [comment]
Great job @sverps 

Oh, since we are "squash&merging" you don't need to push force to the branch to keep the commit history clean. It will also avoid me having tu pull force / delete branch.

Thanks!

**Pull request merged**